tests/testthat/test-remote.R

```diff
@@ -0,0 +1,244 @@
+with_mock_api({
```

Comment:
Are you even using this?

Reply:
Yes, for a few tests — @schloerke and I re-ordered tests to make it clear what's using the mock API and what isn't.